tests/tracking/test_fluent.py

```diff
@@ -433,6 +433,18 @@ def test_start_run_existing_run_deleted(empty_active_run_stack):
             start_run(run_id)
 
 
+def test_start_run_status_running(empty_active_run_stack):
+    mock_run = mock.Mock()
+    mock_run.info.lifecycle_stage = LifecycleStage.DELETED
+
+    run_id = uuid.uuid4().hex
+
+    with mock.patch.object(MlflowClient, "get_run", return_value=mock_run):
+        with pytest.raises(MlflowException):
```

Comment:
```suggestion
        with pytest.raises(MlflowException, match=...):
```

I think we should use the `match` argument ([ref](https://docs.pytest.org/en/stable/reference.html#pytest-raises)) here to avoid catching an incorrect exception.

Reply:
Outdated test